collect_weather_data_from_ghcn_bigquery.py

```diff
@@ -0,0 +1,89 @@
+"""
+Collect Weather Data from GHCN 
+Author: Francisco Vannini 
+Date: 2020-01-29
+
+Requires: 
+Google Project - For more information see : https://cloud.google.com/bigquery/docs/authentication/
+
+Inputs:
+cbc_cleaned_usa_statid.csv - csv file of cleaned cbc data matched with closest station id 
+
+Outputs: 
+flatten_data_from_<DATE>.csv - csv file of Percipitation, Temp low, Temp high and station ID and yer 
+
+"""
+import pandas as pd # Read in unfinished dataset
+from google.cloud import bigquery
+from google.cloud.bigquery.job import LoadJobConfig, WriteDisposition, CreateDisposition
+import pandas as pd
+
+
+# Change this to the PATH of your cbc_cleaned_usa_statid station
+df_shell = pd.read_csv('C:\\Users\\Frankie-Figz\\Downloads\\cbc_cleaned_usa_statid.csv')
+# Find the required dates and station ids 
+station_list = df_shell.loc[:, 'closest_station_id'].unique()
+date_list = df_shell.loc[:,'count_date'].unique()
+# Format the dates and stations ids as to be acceptable to the query that will later be formed
+string_parameter_stations  = ""
+seperator = ' \',\' '
+string_parameter_stations = seperator.join(station_list)
+string_parameter_stations = "(\'" + string_parameter + "\')"
+string_parameter_stations = string_parameter.replace(" ", "")
+string_parameter_dates = ""
+string_parameter_dates = seperator.join(date_list)
+string_parameter_dates = "(\'" + string_parameter_dates + "\')"
+string_parameter_dates = string_parameter_dates.replace(" ", "")
```

Comment:
I would chain this rather than declaring the variable over and over.

```
separator.join(date_list).replace(" ", "") or someting like that
```

Reply:
Yea, you are right, the above could be greatly simplified into less lines. I will look into it